### ISSUE TITLE
Add ResponseCreator method converter from single quote to double quotes

### ISSUE THREAD

**Issue by lordofthejars** [SpectoLabs/hoverfly-java#112]
Currently to return a success you need to skip double quotes:

```java
.willReturn(success("{\"bookingId\":\"1\"}", "application/json"))
```

With small documents is ok but if it is bigger might annoying dealing with it. There are some workarounds like reading from file, but providing a method that allows you to use single quotes as double quotes might simplify things, so for example previous example could be written:

```java
.willReturn(successWithSingleQuotes("{'bookingId':'1'}", "application/json"))
```
Which is quite better to read but also to write.

I'll provide a PR.

**Comment by tommysitu** [comment]
Where does your response body come from, @lordofthejars? If this is a substantial response body, would it be better to load it from a file? Presumably, most  DSL users will pass a POJO to the response creator like so: 
```
.willReturn(success().body(json(myObject)));
```
But maybe I am wrong.

**Comment by tommysitu** [comment]
What you are requesting for is like building inline JSON which is not quite supported by JAVA, and having the response JSON inline with the request is good for reading and reasoning.

Maybe this should be built in Hoverfly itself, @benjih. Having an option for supporting JSON Body with single quote.

**Comment by mogronalol** [comment]
The other alternative is using something like Groovy. Infact this is the one of the reasons we use Groovy / Spock to functional test API's on other projects.

I don't think Hoverfly should support it, as it's invalid JSON. What would happen here is the quotes are converted to double and escaped before being sent across to Hoverfly via it's API.

**Comment by lordofthejars** [comment]
What I was thinking is just a method on Java part to make things more simple in case of using plain Java (not Groovy, ...).

**Comment by lordofthejars** [comment]
@tommysitu yes if the response is so big then it should come from a file, but if it is like 4 or 5 attributes then maybe developer wants to just put it as String.

**Comment by tommysitu** [comment]
Ok, that could be useful. I would recommend you to implement a special `HttpBodyConverter` like this:
```
.willReturn(success().body(jsonWithSingleQuotes("{'bookingId':'1'}")))
```
So it could be used in any methods that accepts the HttpBodyConverter type. 

I found that the [JSONParser](https://github.com/FasterXML/jackson-core/wiki/JsonParser-Features) could do the conversion as well, it has a feature called  `ALLOW_SINGLE_QUOTES `.

It'd be brilliant if you can provide a PR @lordofthejars

**Comment by lordofthejars** [comment]
sure, I'll do it on Monday

**Comment by tommysitu** [comment]
I am going to update the doc and release this new feature tomorrow.

**Comment by tommysitu** [comment]
Fixed in v0.6.2
